### PR TITLE
Baseline average embs

### PR DESCRIPTION
**Pull request by evvahu**
This PR contains a script to generate baseline results for both static and contextualized embeddings.
The phrases are averaged by summing them. 
The contextualized embeddings need a specific dataset to extract the contextualized embeddings (i.e. modifier and head should not be lemmatized but should be kept as they appear in the sentence). 

In this PR, the variable 'phrase' was renamed to 'label' in the StaticRankingDataset for the sake of consistency.

### CODE REVIEW

File: baselines/baseline_word_embeddings.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+def get_average_phrase(data):
+    """
+    :param data: dataset as dataloader
+    :return averages: list of averages of phrases
+    """
+    data_all = next(iter(data))
+    averages = []
+    for m, h in zip(data_all["w1"], data_all["w2"]):
+        averages.append(m.add(h))
```
Comment (Blubberli) [inline baselines/baseline_word_embeddings.py]: This computes the "average phrase" by taking the sum of modifier and head. this a valid baseline but it is not really the "average" if we define "average" as the real average or "centroid" of two vectors (that would require dividing the sum by two)..we can add have two methods, one for getting the sum and one for getting the average

**Review by Blubberli — CHANGES_REQUESTED** [review]
I would say, let's have the two options: either you compute the final vector by taking the sum or by taking the centroid of the single word vectors. the option can be speicified with an additional command line argument...

File: baselines/baseline_word_embeddings.py (hunk 1 above)
Comment (evvahu) [reply baselines/baseline_word_embeddings.py]: done
the option is specified in the config. I think that makes more sense

**Review by Blubberli — APPROVED** [review]
sorry I overlooked that you used a config file instead of single arguments. looks good, this is done [x]

**Pull request merged**